Makefile

```diff
@@ -20,15 +20,15 @@ test:
 	for t in $(TEST_TARGETS); do $(MAKE) $$t || error=1; done; exit $$error
 
 pytest:
-	python2 -m pytest --doctest-modules -vv --junitxml=junit_results.xml --cov-report=html \
+	python3 -m pytest --doctest-modules -vv --junitxml=junit_results.xml --cov-report=html \
```

Comment:
Python3 is Python3.5, that's why the build still fails. All required packages are installed under pip3.7 (presumably)